src/utilities.lisp

```diff
@@ -167,36 +167,34 @@ The result will be a list of cons cells representing half-open intervals (on the
   #-unix
   1)
 
-(let ((prepared? nil)
-      (workers-allotted nil))
-  (defun prepare-for-parallelization (&optional num-workers)
-    "Prepare for parallelization with the correct number of workers scaling with the number of logical cores of your machine.
+(defun prepare-for-parallelization (&optional num-workers)
```

Comment:
Let's make this argument default to`(count-logical-cores)` so we don't have to account for `null`.

Reply:
Yes